lms/db/__init__.py

```diff
@@ -146,6 +155,9 @@ def includeme(config):
     config.registry["sqlalchemy.engine"] = engine
     init(engine)
 
+    if config.registry.settings["dev"]:  # pragma: nocover
+        stamp_db(engine)
```

Comment:
The `alembic` command added above caused problems with database connections on Jenkins. I couldn't figure out why, and debugging it / trying things out was very slow. So I just put the stamping in a separate function and only called it in dev. We don't want to do this in tests, CI or QA and prod anyway, so this is probably a good thing. I don't think it's worth adding unit tests for something that's only used in dev / I think the unit test you'd likely write for this would probably be a bit silly